plugins/backstage-highlights-plugin/src/components/GitInfoCloneField.tsx

```diff
@@ -18,23 +18,25 @@ import React, { useState }  from 'react';
 import { CopyTextButton, LinkButton } from '@backstage/core-components';
 import { Dialog, DialogContent, TextField } from '@material-ui/core';
 import { useEntity } from '@backstage/plugin-catalog-react';
-import { GITHUB_ANNOTATION_PROJECT_SLUG, GITLAB_ANNOTATION_PROJECT_SLUG } from '../util';
+import { GITHUB_ANNOTATION_PROJECT_SLUG, GITLAB_ANNOTATION_PROJECT_SLUG, GITLAB_ANNOTATION_INSTANCE } from '../util';
 import { Alert } from '@material-ui/lab';
 
 const GitInfoCloneDialog = () => {
     const { entity } = useEntity();
 
     const githubAnnotation = entity.metadata.annotations?.[GITHUB_ANNOTATION_PROJECT_SLUG];
     const gitlabAnnotation = entity.metadata.annotations?.[GITLAB_ANNOTATION_PROJECT_SLUG];
+    const gitlabInstance = entity.metadata.annotations?.[GITLAB_ANNOTATION_INSTANCE] ?? 'gitlab.com';
+
 
     let cloneUrl: string | undefined;
-    
+
     if (githubAnnotation) {
-        cloneUrl = `https://github.com/${githubAnnotation}.git`;
-    }
-    if (gitlabAnnotation) {
-        cloneUrl = `https://gitlab.com/${gitlabAnnotation}.git`;
+      cloneUrl = `https://github.com/${githubAnnotation}.git`;
+    } else if (gitlabAnnotation) {
```

Comment:
while it is a correct logic, it is not backward compatible - I mean if someone had before github and gitlab annotations (by accident I guess), then previous logic took gitlab as the more important one. After your change, the github takes precedence. 
Because of that, I would rather keep backward compatibility even if it is not correct, so please remove "else" but leave just "if".